app/src/main/java/org/fossasia/susi/ai/device/deviceconnect/DeviceConnectFragment.kt

```diff
@@ -131,7 +136,9 @@ class DeviceConnectFragment : Fragment(), IDeviceConnectView {
         scanHelp.visibility = View.GONE
         showWifiList = false
         addDeviceButton.visibility = View.GONE
+        scanHelp.visibility = View.GONE
         deviceList.visibility = View.GONE
+        wifi_password_layout.visibility = View.GONE
         wifiList.visibility = View.GONE
```

Comment:
Can't handle visibility for root layout instead of different components?

Reply:
Yes, we can. Here wifiList is the recycler view that is present in the main base layout. So, we need to make it VISIBLE/GONE manually by code according to our need

Reply:
I'm to talking about wifi list, I'm talking about these all layout to handle visibility at once.

Reply:
Nope. We have around 4 layouts integrated into the base layout. The visibility of these layouts depends upon the situation of the connectivity of the device. SO we need to change it according to the situation from the code itself.